projects/backend/controllers/image.ts

```diff
@@ -0,0 +1,9 @@
+import { Request, Response } from 'express'
+import { getRedirect } from '../image'
+export const imageController = (req: Request, res: Response) => {
+    const source = req.params.source
+    const size = req.params.size
```

Comment:
Can we bail out with a nicer error here in case in future `size` doesn't conform to an `ImageSize` - probably unlikely but feels like a cheap bit of safety?

Reply:
Yeah, that's sensible, however users are going to experience that as a 404 from s3 or a missing file in the bundle. (We also need to consider the actual sizes, hence the arbitrary `small` and `notsmall`.)